Dockerfile

```diff
@@ -30,4 +30,5 @@ COPY ./remme ./remme/remme
 COPY ./setup.py ./remme
 RUN pip3 install ./remme
 COPY ./bash/.bashrc /root/.bashrc
-COPY ./tests ./tests
\ No newline at end of file
+COPY ./tests ./tests
+COPY ./protos ./protos
```

Comment:
Do we really need to copy the protos here? This might be needed if we were building the protos right in the Dockerfile, but that is not the case.

Reply:
I did this for executing protos compilation in container during test running .